src/components/FormComponent/FormComponent.js

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react';
+import InputFieldComponent from '../InputFieldComponent/InputFieldComponent'
+
+class FormComponent extends Component {
+	constructor() {
+		super();
+		this.state = {
+			checkbox: false,
+			email: '',
+			firstName: '',
+			lastName: '',
+			password: '',
+		}	
+	}
+
+	handleInput = (e) => {
+		const target = e.target;
+	    const value = target.type === 'checkbox' ? target.checked : target.value;
+	    const name = target.name;
+
+	    console.log("Name: " + name);
+
+	    this.setState({
+	    	[name]: value
+	    });
+
+
+	}
+
+	handleSubmit = (e)  => {
+		e.preventDefault();
+		console.log("Name is: " + this.state.firstName);
+		console.log("Last Name is: " + this.state.lastName);
+		console.log("Password is: " + this.state.password);
+		console.log("Checkbox is: " + this.state.checkbox);
```

Comment:
Please follow the styleguide given here: https://github.com/airbnb/javascript/tree/master/react
It says: Use double quotes for JSX, but for the rest of JavaScript use single quotes